ERROR   2024-08-10 05:25:36,418D:\PycharmProjects\wty_test\logs\error-2024-08-10.log:logControl.py:88 测试
ERROR   2024-08-10 05:25:36,418D:\PycharmProjects\wty_test\logs\error-2024-08-10.log:logControl.py:88 测试
